File: corpus/typescript-runtime-safety-v2/base/report.ts
import { execFileSync } from "node:child_process";

export function renderReport(userName: string, rows: string[]): string[] {
  execFileSync("printf", ["%s", userName]);
  return rows;
}

export function reportTag(requestId: string): string {
  return "request:" + requestId;
}

File: corpus/python-runtime-safety-v2/base/report.py
import subprocess


def render_report(user_name: str, rows: list[str]) -> list[str]:
    subprocess.run(["printf", "%s", user_name], check=True, capture_output=True)
    return rows


def report_tag(request_id: str) -> str:
    return "request:" + request_id
